lalrpop/src/normalize/resolve/mod.rs

```diff
@@ -34,8 +34,19 @@ fn resolve_in_place(grammar: &mut Grammar) -> NormResult<()> {
                        TerminalString::Bare(id) => Some((conversion.span, id, Def::Terminal)),
                    });
 
+        let match_identifiers =
```

Comment:
Seems like a comment would be good here. Just because the rest of LALRPOP is appalingly commented doesn't mean we can't start now. Something like:

```
// Extract all the bare identifiers that appear in the RHS of a `match` declaration.
// Example:
//     match {
//         r"(?)begin" => "BEGIN",
//     } else {
//         r"[a-zA-Z_][a-zA-Z0-9_]*" => IDENTIFIER,
//     }
// This would result in `vec![IDENTIFIER]`.